src/component/youtube/index.tsx

```diff
@@ -77,6 +89,7 @@ function YoutubeButton(props: Props) {
   useEffect(() => {
     const likerButtonInstance = new LikerButton({
       likerId,
+      href: getCanonicalVideoURL(),
```

Comment:
Because we've got a similar function in src/sdk/button.iframe.ts so we need to remove this function?

Reply:
Which function do you mean?

Reply:
getCanonicalVideoURL

Reply:
Saw your context 👍🏽

Reply:
Yes the point of this function is that part where I remove querystring except `v` . Since this is youtube specific I put this function here.
Later we might also want to remove some generic ones like `utm_source`, which might be suitable for handling in button.iframe.ts

Reply:
I don’t recommend to handle particular params like `utm_source` in button.iframe.ts because button.iframe.ts is designed for not only youtube but also other websites.
And the original `button.iframe.ts` also need update for more generic case. But we can discuss the specific piece later in implement vue version later.